TrackQuality/setupEnv.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
+
+SCRIPT_DIR=`dirname ${BASH_SOURCE[0]}`
+STOP=""
```

Comment:
This variable doesn't seem to be used anywhere.

Reply:
This variable has been removed